5.5/contrib/etc/httpdconf.sed

```diff
@@ -1,7 +1,7 @@
 s/^Listen 80/Listen 0.0.0.0:8080/
 s/^User apache/User default/
 s/^Group apache/Group root/
-s%^DocumentRoot "/opt/rh/httpd24/root/var/www/html"%DocumentRoot "/opt/app-root/src"%
+s%^DocumentRoot "/opt/rh/httpd24/root/var/www/html"%#DocumentRoot "/opt/app-root/src"%
```

Comment:
@bparees yep, because I add the DocumentRoot by the template 00-documentroot.conf.template

`DocumentRoot "/opt/app-root/src${DOCUMENTROOT}"`